sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_base.py

```diff
@@ -162,29 +164,27 @@ def send_messages(producer, timeout_time, last_exception, logger):
         Handles sending of event data messages.
         :param ~azure.eventhub._producer.EventHubProducer producer: The producer with handler to send messages.
         :param int timeout_time: Timeout time.
-        :param last_exception: Exception to raise if message timed out. Only used by uamqp transport.
-        :param logger: Logger.
+        :param Exception last_exception: Exception to raise if message timed out. Only used by uamqp transport.
+        :param aAny logger: Logger.
```

Comment:
I think this should be this?
```suggestion
        :param logging.Logger logger: Logger.
```